tests/test_basic_s3.py

```diff
@@ -175,15 +187,23 @@ async def test_result_key_iters(s3_client, bucket_name, create_object):
 
 
 async def test_can_get_and_put_object(
-    s3_client, create_object, bucket_name: str
+    s3_client: aiobotocore.client.AioBaseClient,
+    create_object: Callable,
+    bucket_name: str,
 ):
     key_name = 'foobarbaz'
     await create_object(key_name, body='body contents')
 
     resp = await s3_client.get_object(Bucket=bucket_name, Key=key_name)
-    data = await resp['Body'].read()
-    # TODO: think about better api and make behavior like in aiohttp
-    resp['Body'].close()
+    if httpx and isinstance(resp['Body'], httpx.Response):
+        data = await resp['Body'].aread()
```

Comment:
given body is our own StreamResponse wrapper, i prefer we expose a more sane `read` method to match botocore, their whole thing of `read` vs `aread` is very strange design decision.  This will also reduce the diff